### DISCUSSION TITLE
Integrate prediction output with abghorba/Robinhood-Trading-Bot

### DISCUSSION THREAD

**Discussion by spacepirate0001** [Leci37/TensorFlow-stocks-prediction-Machine-learning-RealTime#5] [Ideas]
It would be really cool to work on integrating output predictions with https://github.com/abghorba/Robinhood-Trading-Bot to automate trading on Robinhood. Let me know if you are interested! Also, can I get added to Telegram group?

**Comment by Leci37** [comment]
the ticket makes me very happy

Once you have reached Model_creation_models_for_a_stock.py, it is because you have executed steps 1 and 2 before.

Try to install , and do some models  You can write me at [leciluis@gmial.com](mailto:leciluis@gmial.com) , so I can help you via telegram.

I am developing another branch with multidimensional preconditions (4h time thresholds) instead of the published single-dimension one (15min time thresholds).
I mean, the multidimensional one makes the operational prediction from all the technical patterns, in a 4h segment (e.g. from 9:00 to 11:00), the mono dimensional one (this document) only takes all the technical patterns given at 11:00.
If you are interested in collaborating in it, please send me an email ,

for more info search "stocks-prediction-multi" in this Readme

**Reply by Leci37** [reply]
If you have problems with installation, let me know. 
I am searching _**collaborators for this project**_. If you have experience and want to collaborate text me on email or github Issues


### Why this stock prediction project ?
Things this project offers that I did not find in other free projects, are:

+ Testing with _**+-30 models**_. Multiple combinations features and multiple selections of models (TensorFlow , XGBoost and Sklearn )
+ Threshold and quality _**models evaluation**_
+ Use _**1k**_ technical indicators
+ Method of best features selection (technical indicators) 
+ _**Categorical target**_ (do buy, do sell and do nothing) simple and dynamic,  instead of continuous target variable
+ Powerful open-market-_**real-time**_ evaluation system 
+ Versatile integration with: Twitter, Telegram and Mail
+ Train Machine Learning model with Fresh today stock data
